docs/intro.rst

```diff
@@ -134,32 +119,78 @@ by importing the package into Python::
 Prerequisite operating system packages
 --------------------------------------
 
-Prerequisite operating system packages can be installed by the pywbem setup
-script using a new command ``install_os``::
+Some of the Python packages used by pywbem have dependencies on operating
+system packages. These operating system packages must be installed before the
+pywbem Python package can be installed. This section describes multiple
+approaches how that can be done, and documents these packages.
+
+* Approach with ``pywbem_os_setup.sh`` script:
+
+  This is the recommended approach, as of pywbem 0.11.
+
+  In this approach, a standalone script that installs the OS-level
+  prerequisites is downloaded and executed.
+
+  * Download the :download:`pywbem_os_setup.sh </pywbem_os_setup.sh>` script
+    (for pywbem version |version|).
+
+  * Execute that script:
+
+    .. code-block:: bash
+
+        $ ./pywbem_os_setup.sh
+
+* Approach with ``install_os`` command of ``setup.py``:
+
+  This approach was introduced in pywbem 0.8 and has been deprecated with
+  pywbem 0.11. The examples are shown for pywbem version 0.8.1.
+
+  * Download the pywbem source archive from Pypi using Pip:
+
+    .. code-block:: bash
+
+        $ pip download --no-deps --no-binary :all: pywbem==0.8.1
+
+    The ability to download packages was introduced in Pip 8.0. When using an
+    older Pip version, you can download the pywbem source archive from its Git
+    repository:
+
+    .. code-block:: bash
+
+        $ curl -o pywbem-0.8.1.tar.gz -L https://github.com/pywbem/pywbem/blob/master/dist/pywbem-0.8/pywbem-0.8.1.tar.gz
+
+    Of course, you can also download it manually from Pypi. Note that only the
+    source archive (``*.tar.gz``) has the capability we use here; the binary
+    archive (``*.whl``) does not have that capability.
+
+  * Extract the pywbem source archive and run the ``install_os`` command of
+    ``setup.py``:
 
-    $ python setup.py install_os
+    .. code-block:: bash
 
-For a number of well-known Linux distributions, the setup script will install
-the packages using the respective installer program (e.g. ``yum`` on RHEL). For
```

Comment:
I am just getting into this now, but Fedora is in process of changing from yum to dnf as installer. Hit this in tests on vagrant saturday.

Reply:
DONE: Added support for dnf installer.